server/common/utils/helpers.js

```diff
@@ -285,6 +287,35 @@ const responseWithComments = comments =>
     };
   });
 
+const responseWithCohortDetails = (doc, userId) => {
+  const {
+    _id,
+    createdAt,
+    description,
+    isArchived,
+    memberIds: membersCollection,
+    name,
+    ownerIds
+  } = doc;
+
+  const isOwner = checkIfArrayContainsUserId(ownerIds, userId);
+  const members = _keyBy(
+    responseWithCohortMembers(membersCollection, ownerIds),
+    '_id'
+  );
+
+  return {
+    _id,
+    createdAt,
+    description,
+    isArchived,
+    isMember: true,
```

Comment:
Is this always true? Maybe you should check if `members` set includes the given `userId` to be sure. Otherwise, rename the function to clearly mark this must be only called by the cohort member `responseWithMembersCohortDetails`, etc..